R/tm_a_pca.R

```diff
@@ -8,8 +8,8 @@
 #' @inheritParams teal::module
 #' @inheritParams shared_params
 #' @param dat (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'   specifying columns used to compute PCA.
-#' @param font_size (`numeric`, optional) Specifies font size.
+#' specifying columns used to compute PCA.
+#' @param font_size (`numeric`) optional specifies font size.
```

Comment:
```suggestion
#' @param font_size (`numeric`) optionally specifies font size.
```
Perhaps optionally would be better to use over here.

Reply:
I think that 

```r
(`numeric`) (optional) specifies font size.
```

would be the most natural in this situation : D

Reply:
or maybe 

```r
(`numeric`) optional, specifies font size.
```

Reply:
I like the second version from Marcin

Reply:
I believe we should standardize this and come to a decision. Do we incorporate it into the sentence directly, or should we use a specific notation? 


 Regarding the use of optionally or (option) create more formats so I think we should finalize it here first. https://github.com/insightsengineering/nestdevs-tasks/issues/52.

Reply:
just put a `,` after optional, so the sentence have logical sense. and it's all good.

Reply:
[c9ccb2c](https://github.com/insightsengineering/teal.modules.general/pull/698/commits/c9ccb2ce4224ac33291abbcedc7611152257fc30)